src/Compilers/Core/CodeAnalysisTest/Diagnostics/SuppressMessageAttributeTests.cs

```diff
@@ -1149,13 +1149,15 @@ public class C2
                 new[] { new ThrowExceptionForEachNamedTypeAnalyzer() },
                 onAnalyzerException: (ex, a, d) => exceptionDiagnostics.Add(d));
 
-            exceptionDiagnostics.Verify(
-                Diagnostic("AD0001", null)
+            var diagnostic = Diagnostic("AD0001", null)
                     .WithArguments(
                         "Microsoft.CodeAnalysis.UnitTests.Diagnostics.SuppressMessageAttributeTests+ThrowExceptionForEachNamedTypeAnalyzer",
                         "System.Exception",
                         "ThrowExceptionAnalyzer exception")
-                    .WithLocation(1, 1));
+                    .WithLocation(1, 1);
+
+            // expect 3 different diagnostics with 3 different contexts.
```

Comment:
because description is now different. since now it contains context info which are different for all 3 cases. (ex, symbol name that was being analyzed)